hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -146,7 +146,7 @@ public class HoodieCompactionConfig extends HoodieConfig {
       .markAdvanced()
       .withDocumentation("Compaction strategy decides which file groups are picked up for "
           + "compaction during each compaction run. By default. Hudi picks the log file "
-          + "with most accumulated unmerged data");
+          + "with most accumulated unmerged data. Support composite strategy by providing, format: strategy_class_name1,strategy_class_name2,...");
```

Comment:
The strategy can be composed with multiple strategies by concatenating the class names with ','.

Reply:
> The strategy can be composed with multiple strategies by concatenating the class names with ','.

done~